src/InvalidArgument.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT;
+
+final class InvalidArgument extends Exception
```

Comment:
As of https://github.com/lcobucci/jwt/pull/462 we should also implement the base exception `Lcobucci\JWT\Exception`